tests/e2e/specs/productDetailPage_spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Product detail page', () => {
+  before(() => {
+    cy.visit('/product/lemare-booties-0778-grey/M0E20000000E0WX');
+  });
+
+  it('Displays a single product details', () => {
+    cy.get('[data-test=pdp-product-title]', { timeout: 20000 })
+      .contains('Booties Lemare grey')
+      .parentsUntil('[data-test=product-description]')
+      .parent()
+      .find('[data-test=quickview-sku]')
```

Comment:
If we are in the PDP there shouldn't be any quickview, the quickview is that summary of the product that appears in the POP. Use a more descriptive name.